README.md

```diff
@@ -217,6 +217,52 @@ var builder = new MessageBuilder("Message Text", "Sender_name", "Recipient_Phone
    var message = builder.Build();
    var result = await client.SendMessageAsync(message);
 ```
+## Sending interactive template messages
+Interactive templates is a newer generation of WhatsApp templates, 
```

Comment:
is => are

And it's not a really a new generation, it is just an added functionality within those templates.
So suggestion (feel free to change it to your own creativity ^^):
`Interactive templates allows you to send templates that include buttons.
For more info please visit https://www.cm.com/app/docs/en/api/business-messaging-api/1.0/index#/whatsapp-template-message`

Reply:
ah sounds better, updated ⚡